homework-1/src/templates/stiky.html

```diff
@@ -0,0 +1,5 @@
+<div class="stiky__footer-texts">
```

Comment:
Why footer? Make text uppercase using css.

If inside `stiky__footer-texts` you have only links with same class name `stiky__footer-text` - you can remove this class name for links.